app/main/views/index.py

```diff
@@ -9,10 +9,28 @@
 from app.formatters import apply_html_class, convert_markdown_template
 from app.main import main
 from app.main.views.pricing import CURRENT_SMS_RATE
-from app.main.views.sub_navigation_dictionaries import features_nav, using_notify_nav
+from app.main.views.sub_navigation_dictionaries import (
+    best_practices_nav,
+    features_nav,
+    using_notify_nav,
+)
 from app.utils.user import user_is_logged_in
 from notifications_utils.url_safe_token import generate_token
 
+feature_best_practices_enabled = (
+    os.getenv("FEATURE_BEST_PRACTICES_ENABLED", "false").lower() == "true"
+)
```

Comment:
See the note above about the use of strings instead of boolean values.  We also shouldn't read from the environment variable directly at this point as we've already done that and it's now available in the app config. 🙂   

When making that switch to account for these things, we could then eliminate this bit of code and the check below becomes even simpler. 🙂 

```suggestion
```